scalableminds/webknossos
# Fix displayed segment

This PR adds looking for segmentation ids in higher resolutions when the current resolution has no data and render missing data black is disabled for the `segmentation tab`/`mapping_info_view`.


### URL of deployed dev instance (used for testing):
- _.webknossos.xyz

### Steps to test:
- It is best to test this on a local machine as manipulating datasets is easier.
- Create a dataset 
   - with at least one color layer with the resolutions 1-1-1, 2-2-2, 4-4-4 and so on
   - and with one segmentation layer that does not have the resolution 1-1-1.
- Open the dataset in view mode, disable render missing data black and open the segmentation tab
- Now zoom into mag 1
- The shown segmentation should be the segmentation from mag 2.
- The displayed segment ids in the segmentation tab should match the displayed data as well as there should be a new entry in the table of the tab displaying the resolution that is used to determine the segmentation ids.

### Issues:
- fixes #4431

------
